### ISSUE TITLE
1.3.5 unable to serve GUI

### ISSUE THREAD

**Issue by billyjanitsch** [stefanwichmann/kelvin#116]
Hi, the latest version released on Docker Hub (1.3.5) seems to have an issue serving the GUI. The back-end appears to be working, but when trying to access the GUI, the server does not respond, and the following error appears in the container log:

```
2023/03/30 08:40:40 http: panic serving 172.17.0.1:36318: html/template: pattern matches no files: `gui/template/dashboard.html` 
goroutine 56 [running]:
net/http.(*conn).serve.func1()
	/opt/hostedtoolcache/go/1.20.2/x64/src/net/http/server.go:1854 +0xbf
panic({0x88b860, 0xc000216200})
	/opt/hostedtoolcache/go/1.20.2/x64/src/runtime/panic.go:890 +0x263
html/template.Must(...)
	/opt/hostedtoolcache/go/1.20.2/x64/src/html/template/template.go:368
main.dashboardHandler({0xace040?, 0xc000214720}, 0xc00018c300?)
	/home/runner/work/kelvin/kelvin/webinterface.go:72 +0x216
[...]
```

**Comment by stefanwichmann** [comment]
Can confirm. Apparently the GUI files are missing in the image... will look into it

**Comment by stefanwichmann** [comment]
Fix released with version 1.3.6
